Source/CBPeripheralDelegateWrapper.swift

```diff
@@ -57,6 +57,12 @@ import RxSwift
     var rx_peripheralReadyToSendWriteWithoutResponse: Observable<Void> {
         return peripheralIsReadyToSendWriteWithoutResponseSubject
     }
+    
+    @available(OSX 10.13, iOS 11, *)
+    var rx_didOpenL2CAPChannel: Observable<(CBL2CAPChannel?, Error?)> {
+        return peripheralDidOpenL2CAPChannelSubject
+            .map {($0.0 as? CBL2CAPChannel, $0.1)}
```

Comment:
Do we need to map it? I think that we get from `peripheral: didOpen: error:` an CBL2CAPChannel so if `PublishSubject` will receive in `onNext` this type and not `Any?` we could get rid off this map.